memory/table_editor.go

```diff
@@ -77,7 +77,7 @@ func (t *tableEditor) Insert(ctx *sql.Context, row sql.Row) error {
 	if added {
 		pkColIdxes := t.pkColumnIndexes()
 		vals := make([]interface{}, len(pkColIdxes))
-		for _, i := range pkColIdxes {
+		for i, _ := range pkColIdxes {
```

Comment:
Just use the simple form if all you want is indexes:

`for i := range pkColIdxes`